### DISCUSSION TITLE
[Feature] increase sensitivity when swiping

### DISCUSSION THREAD

**Discussion by Roy416** [immich-app/immich#4005] [Feature Request]
### The feature

When swiping left or right through the photos/videos, I've noticed the sensitivity to be lower than apps such as Instagram, Apple Photos, and Google Photos. I think increasing the sensitivity would increase the perception of the app being more responsive.  

Using an iPhone 12

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by alextran1502** [comment]
This is actually a bug that we are planning to fix

**Comment by Roy416** [comment]
Oh awesome! Would that include changing the swiping down/up gesture behavior as well? I was going to do a little more testing before making a post

**Reply by NicholasFlamy** [reply]
Check out #5895
I think it addresses what this is about.
